src/Germany/Mews.Fiscalizations.Germany/V2/FiskalyClient.cs

```diff
@@ -76,7 +76,7 @@ public Task<ResponseResult<Tss>> GetTssAsync(AccessToken token, Guid tssId)
         public Task<ResponseResult<IEnumerable<Tss>>> GetAllTSSsAsync(AccessToken token)
         {
             return Client.GetResponseAsync<Dto.MultipleTssResponse, IEnumerable<Tss>>(
-                endpoint: $"tss",
+                endpoint: $"tss?states=CREATED&states=INITIALIZED&states=UNINITIALIZED",
```

Comment:
Don't like the hardcoding. Let's create a flagenum and set a default parameter value?